dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+var $taskList = document.querySelector('.task-list');
+
+function handleClick(event) {
+  console.log('event.target: ');
+  console.log(event.target);
+  console.log('Event.target tagname: ', event.target.tagName);
+  if (event.target.matches('Button')) {
```

Comment:
Please revise this to be using the `tagName` property covered in the exercise rather than the `matches` method

Reply:
![image](https://user-images.githubusercontent.com/51938797/112529318-5eb28000-8d62-11eb-91bf-eb7b56028c07.png)

Updated with method advocated by the exercise. Sorry I looked in the Blog also.